### ISSUE TITLE
Find plagiarize project

### ISSUE THREAD

**Issue by liuhuiAndroid** [chrisbanes/accompanist#156]
https://github.com/skydoves/Landscapist
This project copy your code and not marked.

**Comment by chrisbanes** [comment]
That project does indeed look very similar. The new library name and it's recent [`ImageLoad`](https://github.com/skydoves/Landscapist/blob/master/landscapist/src/main/java/com/skydoves/landscapist/ImageLoad.kt) function look _especially_ similar. 

That said, I hope people **do** copy from this project and use it as a reference for how to write Compose libraries... just maybe not a copy to do the same thing 😆 

/cc @skydoves

**Comment by skydoves** [comment]
Oh, That was my huge mistake not to add [references](https://github.com/skydoves/Landscapist#reference-repository) related to this repository!

I definitely agree that I referenced this repository before design the library. And I had much interest in the `ImageLoad` function and how this library was structured especially. I've learned a lot of things from chrisbanes's codes and I wanted to experiment little different kinds of things related to animations.

And I appreciate so much chrisbanes for saying kindly about that. 🙏
I'm your big fan and have been respected you for a long time!
